### PR TITLE
[issue 250] - Text wrapping for different languages

### PR DESCRIPTION
**Pull request by DeeJayElly**
[issue 250] - Text wrapping for different languages

### CODE REVIEW

File: source/features/i18n/translations/de.json [hunk 1]
```diff
@@ -73,7 +73,7 @@
     "cardanoFoundationYoutube": "Cardano Foundation Youtube",
     "cardanoReddit": "Cardano Reddit",
     "cardanoText": "!!!Cardano is a software platform ONLY and does not conduct any independent diligence on or substantive review of any blockchain ,digital currency, cryptocurrency or associated technology. Your use of this website and software is entirely at your own risk, and this website is provided on an \"as is\" basis and for reference only.",
-    "cardanoSL": "!!!Cardano SL Source Code",
+    "cardanoSL": "Cardano SL Source Code",
```
Comment (a-rukin) [inline source/features/i18n/translations/de.json]: @DeeJayElly is this a real translation? 
GTranslate says: "Cardano SL Quellcode" 🤷‍♂

Comment (DominikGuzei) [reply source/features/i18n/translations/de.json]: Im the German guy here … and it's not a proper translation 😛
Let's refactor this to "Cardano-SL Quellcode"

Comment (DominikGuzei) [reply source/features/i18n/translations/de.json]: I fixed this 👍

**Comment by a-rukin** [comment]
@DominikGuzei did you change something?

**Comment by DominikGuzei** [comment]
@a-rukin yeah i fixed the German translation of "cardano sl"

**Pull request merged**